packages/roosterjs-editor-plugins/lib/pluginUtils/DragAndDropHelper.ts

```diff
@@ -6,31 +6,47 @@ import DragAndDropHandler from './DragAndDropHandler';
  * @internal
  * Compatible mouse event names for different platform
  */
-interface MouseEventNames {
+interface MouseEventInfo {
     MOUSEDOWN: string;
     MOUSEMOVE: string;
     MOUSEUP: string;
+    getPageXY: (e: Event) => number[];
```

Comment:
you can return [number,number] to limit the return value to only has two items

Reply:
Updated! Thanks